woocommerce/woocommerce-admin
# Remove the Onboarding Levels API

This PR removes the "Onboarding Levels API" that was originally created for an old site of onboarding designs. They are no longer in use.

To Test:
* Run `phpunit` and make sure all tests pass.
* Load the dashboard and make sure there are no fatals.
